i18n/en-US.properties

```diff
@@ -316,7 +316,7 @@ be.sidebarVersions.empty = No prior versions are available for this file.
 be.sidebarVersions.preview = Preview
 # Label for the version promote action.
 be.sidebarVersions.promote = Make Current
-# Label for the version delete action.
+# Label for the version restore action.
```

Comment:
@jstoffan Let me know if this looks alright to you. Seems like a commit might have just missed a translations build.